txbridge/src/main/java/org/jboss/jbossts/txbridge/outbound/BridgeXAResource.java

```diff
@@ -70,7 +71,7 @@
      * @param externalTxId the parent JTA transaction identifier.
      * @param bridgeWrapper the control for the subordinate WS-AT transaction.
      */
-    public BridgeXAResource(Uid externalTxId, BridgeWrapper bridgeWrapper)
+    public BridgeXAResource(Transaction externalTxId, BridgeWrapper bridgeWrapper)
```

Comment:
@mmusgrov so the back-ward compatible constructor should throw some `IllegalStateException` when is called? Or how to proceed?

I mean it would be good to fail fast than leave the runtime fail to some further times where tracking could be harder. What do you think?